examples/vdb/hana/bookmark/main.tf

```diff
@@ -0,0 +1,140 @@
+terraform {
```

Comment:
We should add a simple description to the top of these examples files. I think we wrote some during our original implementation?

Suggestion:
```
/**
* Summary: This template showcases the properties available when provisioning an SAP Hana database from a DCT bookmark.
*/
```

Reply:
yes will do.